---
layout: archive
title: "Tutorials and useful links"
permalink: /links/
author_profile: true
---


(Updated 30/07/2021)

General genomics (sequencing, lib prep, extraction)
------
* [Excellent tutorials on every flavor of genomics](https://bioinformaticsworkbook.org/)
* [Info on Pacbio's Sequel II](https://www.pacb.com/wp-content/uploads/Overview-Sequel-Systems-Application-Options-and-Sequencing-Recommendations.pdf)
* [Extracting long-read DNA for Pacbio](https://extractdnaforpacbio.com/)
* Starting with Genomics, unix and programming? Look no further, Ming Tang has compiled [a comprehensive list of resources](https://github.com/crazyhottommy/getting-started-with-genomics-tools-and-resources)
* Sequence [file formats](https://snipcademy.com/sequence-file-formats)
* Catalogue of [long-read tools](https://long-read-tools.org/analysis.html)
* Tons of [one liners to modify genomic files!](https://github.com/crazyhottommy/bioinformatics-one-liners/blob/master/README.md)

Population Genomics
------
* PCA, STRUCTURE, SNAPP - [Some of my scripts](https://github.com/jcerca/population_genomics)
* ABBA-BABA [synthesis and plot](https://nbviewer.jupyter.org/github/dereneaton/ipyrad/blob/master/tests/cookbook-abba-baba.ipynb)
* List of tutorials in speciation genomics by [Mark Ravinet and Joana Meier](https://speciationgenomics.github.io/)
* List of programs in populations genomics [using genomic-level data](http://methodspopgen.com/)
* List of tutorials included [Evomics course on Speciation and Population Genomics 2016 and 2018 tutorials - including ABBA-BABA, FasSimCoal, PCA, STRUCTURE](http://evomics.org/learning/population-and-speciation-genomics/)
* SNP-filtering tutorial [by Jonathan Puritz](https://www.ddocent.com/filtering/)
* Book on population genomics with [code and exercises](https://link.springer.com/book/10.1007/978-1-0716-0199-0)
* Have not yet tested, but maybe a practical way to [run D-statistics from a VCF](https://www.biorxiv.org/content/10.1101/634477v2)
* Variant-calling [tutorial](https://bioinformaticsworkbook.org/dataAnalysis/VariantCalling/variant-calling-index)
* Needing some help with [GATK for non-model organisms?](https://evodify.com/gatk-in-non-model-organism/)
* Mark Ravinet has a three part tutorial on how to detect selection! See [part 1 here](https://markravinet.github.io/inferring_selection_part1.html), [part 2 here](https://markravinet.github.io/inferring_selection_part2.html), and [part 3 here](https://markravinet.github.io/inferring_selection_part1.html).
* Brilliant tutorial on [hybridization/introgression](https://github.com/millanek/tutorials/tree/master/analysis_of_introgression_with_snp_data) 
* Brilliant tutorial on [low-coverage data / ANGSD](https://github.com/mfumagalli/ngsTools/blob/master/TUTORIAL.md)

Comparative Genomics
------
* Comprative genomics [tutorials](https://bioinformaticsworkbook.org/dataAnalysis/ComparativeGenomics/compGenomics_index)
* Long-read sequencer [comparison](https://github.com/rrwick/Long-read-assembler-comparison)
* CAFE tutorial [for gene family evolution](https://iu.app.box.com/v/cafetutorial-files)
* [Repeat annotation](https://bioinformaticsworkbook.org/dataAnalysis/ComparativeGenomics/Repeats_index) in genomes
* Trying to estimate plody-level with the [GenomeScope-Smudgeplot combo](https://github.com/tbenavi1/genomescope2.0), but are getting weird results? [Kamil Jaron has a bunch to show you](https://kamilsjaron.github.io/peculiar-genomic-observations/)
* Handling [GTF, GFF, GFF3 files](https://github.com/NBISweden/AGAT/)
* How to [run and interpret OrthoFinder](https://davidemms.github.io/menu/tutorials.html)
* Retrieving [Homologous Chromossomic regions](http://chibba.pgml.uga.edu/mcscan2) and [plotting it](http://synvisio.github.io)
* Excellent tutorial on [genome repeat and TE annotation](https://www.biostars.org/p/411101/)
* Promising [list of tutorials](https://genomevolution.org/wiki/index.php/Tutorials#Finding_Inversions) with CoGe to find features in your genome.
* Two ways of making better use of RepeatMasker to build RepeatLibraries. See [here](http://weatherby.genetics.utah.edu/MAKER/wiki/index.php/Repeat_Library_Construction-Advanced) and [here](https://www.biostars.org/p/411101/).
* This [program looks like an easy way to look for synteny!](http://www.agcol.arizona.edu/software/symap/doc/Tour.html)

Phylogenomics
------
* List of tutorials on phylogenetic inference [by Michael Matschiner](https://github.com/mmatschiner/tutorials)
* Tutorials on phylogenomics included in the [Evomics course 2016 and 2018 tutorials](http://evomics.org/learning/phylogenetics/)
* Really good [tree handling and plotting examples](https://yulab-smu.github.io/treedata-book/)
* Phylogenies with vcftools? Maybe try [SNPphylo](http://chibba.pgml.uga.edu/snphylo/)

RNA-Seq
-------
* Mind-blowing list of RNA-seq [methods, tools, pitfalls, tutorials, pipelines, packages](https://github.com/crazyhottommy/RNA-seq-analysis)
* RNA-Seq [tutorial](https://github.com/crazyhottommy/rnaseq_tutorial)
* RNA-Seq ['genome browser'](http://xena.ucsc.edu/welcome-to-ucsc-xena/)
* RNA-Seq [tutorial](https://bioinformaticsworkbook.org/dataAnalysis/RNA-Seq/RNA-SeqIntro/RNAseq-intro)
* RNA-Seq [basics, and pipeline overview](https://rnaseq.uoregon.edu/)

Methylation
------
* Comprehensive list of DNA-Methylation[methods, tools, pitfalls, tutorials, pipelines, packages](https://github.com/crazyhottommy/DNA-methylation-analysis)

Evolutionary Biology
------
* Easily broken down [statistics for biologists!](https://www.nature.com/collections/qghhqm/pointsofsignificance)

Data Visualization (R, python)
------
* Pathwork, an interesting way to [combine plots](https://patchwork.data-imaginist.com/)
* Something I want to explore better, [plotly for R and python](https://plot.ly/r/)
* The usual suspect, [ggplot2](http://t-redactyl.io/tag/ggplot2.html)

Career-oriented
------
* Establishing a laboratory book [for new PIs](https://twitter.com/samuelmehr/status/1139733291899080705)
